tests/_support/Page/Framework.php

```diff
@@ -1093,4 +1096,64 @@ public function iUploadTheItemTypeCASEFile()
 
         return $this;
     }
+
+     /**
+     * @When /^I see the Export Comment button$/
+     */
+    public function iSeeExportCommentButton()
```

Comment:
the functions related to comments should be moved to a different `Comment` class